src/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs

```diff
@@ -54,9 +54,9 @@ public abstract partial class PipeReader
         /// Returns a <see cref="Stream"/> that wraps the <see cref="PipeReader"/>.
         /// </summary>
         /// <returns>The <see cref="Stream"/>.</returns>
-        public virtual Stream AsStream()
+        public virtual Stream AsStream(bool leaveOpen = false)
```

Comment:
Since this method has API documentation present, please add the doc comment for the leaveOpen parameter.